src/stdio.jl

```diff
@@ -52,6 +84,43 @@ function watch_stream(rd::IO, name::AbstractString)
     end
 end
 
+function label_timed_task(name)
+    if verbose::Bool && !haskey(task_local_storage(), :IJulia_task)
+        label = "send $name task"
+        task_local_storage(:IJulia_task, label)
+        @vprintln("$label labelled")
+    end
+end
+
+function send_stdout(t::Timer)
+    name = "stdout"
+    label_timed_task(name)
+    send_stream(name)
+end
+
+function send_stderr(t::Timer)
```

Comment:
However, I don't think this is the right function to set `task_local_storage` from, as this should normally be called from a function where that has already been set.

So, you might as well eliminate this function entirely and call `send_stream` directly.